src/lib/token.h

```diff
@@ -91,7 +89,7 @@ struct token {
             sealobject sealobject;
         } esysdb; /* esysdb */
         struct {
-            FAPI_CONTEXT *ctx;
+            void *fapi_ctx;
```

Comment:
The reason I had tpm_ctx pointer is to keep the TPM types minimized from the rest of the system, hence uint32_t vs ESYS_TR